### PR TITLE
use Armory easy16 backup code

### PR DESCRIPTION
**Pull request by sergey-chernikov**
This is to allow wallet import in new terminal arch branch, as new and old export routines produce different output.
Use only with https://github.com/BlockSettle/common/pull/2126

### CODE REVIEW

File: BlockSettleSigner/main.cpp [hunk 1]
```diff
@@ -336,7 +336,7 @@ static int QMLApp(int argc, char **argv
 
          terminalConnectionTimer.setSingleShot(true);
          //BST-2786
-         terminalConnectionTimer.setInterval(std::chrono::milliseconds{ 5000 });
+         terminalConnectionTimer.setInterval(std::chrono::milliseconds{ 25000 });
```
Comment (Ation) [inline BlockSettleSigner/main.cpp:339]: Is this intended?

File: BlockSettleSigner/EasyEncValidator.cpp [hunk 2]
```diff
@@ -25,19 +25,19 @@ QValidator::State EasyEncValidator::validate(QString &input, int &pos) const
...
       }
       const char c = tmpInput.at(i).toLatin1();
-      if ((allowedChars.find(c) == allowedChars.end())) {
+/*      if ((allowedChars.find(c) == allowedChars.end())) {
          input = newInput;
          pos = newPos;
          setStatusMsg(invalidMsgTmpl_.arg(name_));
          return QValidator::State::Invalid;
-      }
+      }*/
```
Comment (Ation) [inline BlockSettleSigner/EasyEncValidator.cpp]: Can you please remove input validation, if it is not used?

File: BlockSettleSigner/main.cpp (hunk 1 above)
Comment (sergey-chernikov) [reply BlockSettleSigner/main.cpp:339]: Yes, this helps to run signer properly with 2 wallets with 3k+ addresses in Windows debug. Lower delay values lead to signer TCP error. Scott was also complaining about that on Linux release with D8s prod wallet.

File: BlockSettleSigner/EasyEncValidator.cpp (hunk 2 above)
Comment (sergey-chernikov) [reply BlockSettleSigner/EasyEncValidator.cpp]: Input validation is used, I just only commented out the validation of easy16 chars which wasn't working for some reason with new code. Left it commented (not removed) to return to it later after the list will be available from Armory code.

Comment (Ation) [reply BlockSettleSigner/EasyEncValidator.cpp]: Can you add some text marker like TODO, XXX or anything else that will explain why code is left and what should be done and how and when.

File: BlockSettleSigner/main.cpp (hunk 1 above)
Comment (Ation) [reply BlockSettleSigner/main.cpp:339]: I think that we are going a bit wrong path. What if there is even bigger wallet and slower PC? This must be a timer for establishing network connection, not for a complete loading of wallets. Just thoughts.

Comment (sergey-chernikov) [reply BlockSettleSigner/main.cpp:339]: Agree, I wasn't trying to fix it once and for all - this is just a workaround for an average comfort use. We can return to fixing it at signer arch refactoring (with bus and adapters).

File: BlockSettleSigner/EasyEncValidator.cpp (hunk 2 above)
Comment (sergey-chernikov) [reply BlockSettleSigner/EasyEncValidator.cpp]: added

**Pull request merged**